introduction/site_survey.md

```diff
@@ -0,0 +1,353 @@
+liquid-cooled cabinets, device diagrams for switches and nodes in the cabinets, list of source and destination of every HSN cable,
+list of source and destination of every cable connected to the spine switches, list of source and destination of every cable connected
+to the NMN, list of source and destination of every cable connected to the HMN. list of cabling for the KVM, and routing of power to the PDUs.
+
+The installation of CSM software requires that the SHCD be available.  Some information will be manually collected from the SHCD, but some of the tabs can be extracted into CSV formatted files to be fed to automatic configuration tools.
+
+<a name="csi_command_line_configuration_payload"></a>
+## 7. `csi` Command Line Configuration Payload
+
+This information from a site survey can be given to the `csi` command as command line arguments.
+The information is shown here to explain what data is needed.
+
+The air-cooled cabinet is known to `csi` as a `river` cabinet. The liquid-cooled cabinets are either
+`mountain` or `hill` (if a TDS system).
+
+| CSI option | Example | Information | Site Value |
+| --- | --- | --- | --- |
+| --bootstrap-ncn-bmc-user | root | Administrative account for the management node BMCs | | 
+| --bootstrap-ncn-bmc-pass | changeme | Password for `bootstrap-ncn-bmc-user` account | | 
+| --system-name | eniac | Name of the HPE Cray EX system | | 
+| --mountain-cabinets | 4 | Number of Mountain cabinets, but this could also be in `cabinets.yaml` | | 
+| --starting-mountain-cabinet | 1000 | Starting Mountain cabinet | | 
+| --hill-cabinets | 0 | Number of Hill cabinets, but this could also be in `cabinets.yaml` | | 
+| --river-cabinets | 1 | Number of River cabinets, but this could also be in `cabinets.yaml` | | 
+| --can-cidr | 10.103.11.0/24 | IP subnet for the CAN assigned to this system | | 
+| --can-external-dns | 10.103.11.113 | IP address on CAN for this system's DNS server | | 
+| --can-gateway | 10.103.11.1 | Virtual IP address for the CAN (on the spine switches) | | 
+| --can-static-pool | 10.103.11.112/28 | MetalLB static pool on CAN | | 
+| --can-dynamic-pool | 10.103.11.128/25 | MetalLB dynamic pool on CAN | | 
+| --hmn-cidr | 10.254.0.0/17 | Override the default cabinet IPv4 subnet for River HMN | | 
+| --nmn-cidr | 10.252.0.0/17 | Override the default cabinet IPv4 subnet for River NMN | | 
+| --hmn-mtn-cidr | 10.104.0.0/17 | Override the default cabinet IPv4 subnet for Mountain HMN | | 
+| --nmn-mtn-cidr | 10.100.0.0/17 | Override the default cabinet IPv4 subnet for Mountain NMN | | 
+| --ntp-pools | time.nist.gov | External NTP pool(s) for this system to use | | 
+| --site-domain | dev.cray.com | Domain name for this system | | 
+| --site-ip | 172.30.53.79/20 | IP address and netmask for the PIT node `lan0` connection | | 
+| --site-gw | 172.30.48.1 | Gateway for the PIT node to use | | 
+| --site-nic | p1p2 | NIC on the PIT node to become `lan0` | | 
```

Comment:
Don't we also allow multiple NICs to be specified here?

Reply:
Not that I'm aware of.   Do we allow lan0 to be bonded?  That is pointless when configuring the PIT node, but has bearing later for lan0 on ncn-m001.